source/tyr/tyr/binarisation.py

```diff
@@ -602,7 +602,42 @@ def stops2mimir(self, instance_config, input, job_id=None, dataset_uid=None):
     try:
         res = launch_exec('stops2mimir', argv, logger)
         if res != 0:
-            raise ValueError('stops2mimir failed')
+            # Do not raise error because that it breaks celery tasks chain.
+            # stops2mimir have to be non-blocking
+            logger.error('stops2mimir failed')
```

Comment:
I like the idea of not failing if stop2mimir fail, but I don't like that noboby will see the failure.
It should not be done in this PR, but we will have to report failure like this some way.

Reply:
Okay, so I add a TODO for future PR.
:white_check_mark: